spec.html

```diff
@@ -35418,16 +35814,25 @@ <h1>
         </emu-alg>
         <emu-grammar>Atom :: `.`</emu-grammar>
         <emu-alg>
-          1. Let _A_ be the CharSet of all characters.
+          1. Let _A_ be the CharSet returned by ! GetAllCharacters().
           1. If _DotAll_ is not *true*, then
             1. Remove from _A_ all characters corresponding to a code point on the right-hand side of the |LineTerminator| production.
           1. Return ! CharacterSetMatcher(_A_, *false*, _direction_).
         </emu-alg>
         <emu-grammar>Atom :: CharacterClass</emu-grammar>
         <emu-alg>
           1. Let _cc_ be CompileCharacterClass of |CharacterClass|.
-          1. Return ! CharacterSetMatcher(_cc_.[[CharSet]], _cc_.[[Invert]], _direction_).
+          1. If _UnicodeSets_ is *false* or if every element of _cc_.[[CharSet]] consists of a single character, return ! CharacterSetMatcher(_cc_.[[CharSet]], _cc_.[[Invert]], _direction_).
+          1. Assert: _cc_.[[Invert]] is *false*.
+          1. Let _D_ be the Disjunction containing:
+            1. an Alternative for each string in _v_ whose length is greater than 1, with longer strings before shorter strings.
+            1. an Alternative for the CharSet containing all and only the single characters in _v_ (that is, all and only the strings of length 1).
+            1. an Alternative for the empty String, if it is in _v_.
```

Comment:
Change step numbers to asterisks (bullets)?

Reply:
[The match order is important](https://github.com/tc39/proposal-regexp-set-notation#whats-the-match-order-for-character-classes-containing-strings), so I think we want to stick to a numbered list.